pyqmc/optimize_ortho.py

```diff
@@ -50,7 +50,11 @@ def collect_overlap_data(wfs, configs, pgrad):
     ) / len(ref)
 
     dppsi = pgrad.transform.serialize_gradients(wfs[-1].pgradient())
-    node_cut, f = pgrad._node_regr(configs, wfs[-1])
+    d2 = 0.0
```

Comment:
can you get this from the line below? 
`    dat = pgrad.avg(configs, wfs[-1], weight[-1])`

Reply:
I haven't figured out how. pgrad.avg averages over configs, but we need that axis for averaging overlap_gradient

Reply:
you can just call it and not do the averaging, then. The only issue is that then the matrix becomes very large.